packages/zen-bull-netting/src/ZenBullNetting.sol

```diff
@@ -227,12 +230,29 @@ contract ZenBullNetting is Ownable, EIP712, FlashSwap {
         uint256 indexed receiptIndex
     );
     event SetBot(address bot);
-    /// @dev shared events with the NettingLib for client side to detect them
-    event DepositAuction(address indexed trader, uint256 indexed bidId, uint256 quantity);
-    event WithdrawAuction(
-        address indexed trader, uint256 indexed bidId, uint256 quantity, uint256 price
+    event DepositAuction(
+        uint256 wethDeposited, uint256 crabAmount, uint256 clearingPrice, uint256 depositsIndex
     );
+    event WithdrawAuction(uint256 zenBullWithdrawn, uint256 clearingPrice, uint256 withdrawsIndex);
     event CancelNonce(address trader, uint256 nonce);
+    /// @dev shared events with the NettingLib for client side to detect them
+    event TransferWethFromMarketMakers(
+        address indexed trader, uint256 quantity, uint256 wethAmount, uint256 clearingPrice
```

Comment:
Remaining amount like for TransferOsqthToMarketMakers/TransferOsqthFromMarketMakers/TransferWethToMarketMaker. Not sure if useful but we have for the others.